package.json

```diff
@@ -13,7 +13,7 @@
     "wp_org_slug": "woocommerce-payments"
   },
   "scripts": {
-    "postinstall": "composer install",
+    "postinstall": "composer install && (npx husky install || true)",
```

Comment:
Hey @marcinbot, I had a problem with this renovate PR. Husky hooks stopped working after I dropped node_modiules and installed all deps. So I came with this construct which solve the problem.

Are there any better alternatives you are aware of?

Reply:
This construct forces Husky to recreate hooks.

Reply:
When I'm on the older husky version (and older hooks versions) and do `rm -rf node_modules && npm install` the hooks are not updated. So I getting broken hooks and git skips them (mean I can commit broken things freely).

As a developer, I'd not drop hooks when starting a new branch.
Alternatively I can revert the postinstall script changes and poke team via Slack.

Not sure what will work best for us...